1.0/requirements.txt

```diff
@@ -1,3 +1,3 @@
-flask==1.0.3
+flask==1.0.4
 gunicorn==19.9.0
```

Comment:
Gunicorn has gone up to v20.0.4. We should evaluate the `CHANGELOG` and determine the feasibility of upgrading it here as well.

Reply:
The biggest breaking change for v20.0.x is dropping support for Python 2.x. We can do this upgrade since we're dropping support as well.